src/main/kotlin/io/lettuce/core/api/coroutines/RedisSortedSetCoroutinesCommands.kt

```diff
@@ -173,6 +173,34 @@ interface RedisSortedSetCoroutinesCommands<K : Any, V : Any> {
      */
     suspend fun zcount(key: K, range: Range<out Number>): Long?
 
+    /**
+     * Computes the difference between the first and all successive input sorted sets.
+     *
+     * @param keys the keys.
+     * @return List<V> array-reply list of elements.
+     * @since 6.2
+     */
+    suspend fun zdiff(vararg keys: K): Flow<V>
```

Comment:
Actually, methods that return `Flow` should not be `suspendable` because they have no suspension points.
Looks like we have to fix Kotlin API generator to prevent such behavior.

Reply:
You were right. Fixed. Thanks.